### PR TITLE
Fix error with css helper from styled-components

### PR DESCRIPTION
**Pull request by hieuhlc**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x]  Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: #34545 
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by ArjanJ — APPROVED** [review]
Looks good, thanks for this!

**Comment by hieuhlc** [comment]
This PR shouldn't be merged now, I've found one more issue with it and will push the fix shortly

**Comment by hieuhlc** [comment]
Now it's ok to merge

File: types/styled-theming/index.d.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
 // Type definitions for styled-theming 2.2
 // Project: https://github.com/styled-components/styled-theming#readme
 // Definitions by: Arjan Jassal <https://github.com/ArjanJ>
+//                 Hieu Ho <https://github.com/hieuhlc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.7
+// TypeScript Version: 3.4
```
Comment (uniqueiniquity) [inline types/styled-theming/index.d.ts]: This change will require all of your package's consumers to use TS 3.4 or above.
Does this change truly make the package incompatible with earlier versions than that?

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/styled-theming/index.d.ts (hunk 1 above)
Comment (hieuhlc) [reply types/styled-theming/index.d.ts]: Version 2.7 didn't work so I changed it to match my current TS version. I fixed it. It only needs to catch up with `styled-components` (version 2.9)
Thank you

**Review by uniqueiniquity — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**